src/ripple/app/ledger/LedgerMaster.h

```diff
@@ -39,6 +39,7 @@
 #include <ripple/protocol/Protocol.h>
 #include <ripple/protocol/RippleLedgerHash.h>
 #include <ripple/protocol/STValidation.h>
+#include <boost/optional.hpp>
```

Comment:
I _think_ this isn't needed here and can move to the **`LedgerMaster.cpp`** file.

Reply:
A bunch of function declarations return boost::optional in this file.